### ISSUE TITLE
homepage- Missing form label

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#89]
While using the WAVE tool, I was alerted to there being a missing form label. 

This is the code it's saying has an error: `<textarea id="g-recaptcha-response-100000" name="g-recaptcha-response" class="g-recaptcha-response" style="width: 250px; height: 40px; border: 1px solid rgb(193, 193, 193); margin: 10px 25px; padding: 0px; resize: none; display: none;">`

I think this has to do with the search form, but we can discuss this next week.

**Comment by kimisgold** [comment]
Hmm, this looks like markup from a recaptcha form that shouldn't be there. I recently applied some changes to how the recaptcha form is handled; can you double check to see if you're still seeing this from the home page?

**Comment by IdaAkiwumi** [comment]
This is what shows up now after refreshing the page and using the tool again.
<img width="450" alt="Screen Shot 2020-10-05 at 11 25 26 AM" src="https://user-images.githubusercontent.com/618307/95099255-a8000c00-06fd-11eb-97c4-827de5c34280.png">

And this is what the WAVE tool is saying about this. Hope this helps:

![Screen Shot 2020-10-05 at 11 27 15 AM](https://user-images.githubusercontent.com/618307/95099373-c6660780-06fd-11eb-81a4-a6e34cafe8b9.png)

It's also still highlighting that code: 
![Screen Shot 2020-10-05 at 11 29 30 AM](https://user-images.githubusercontent.com/618307/95099652-2c528f00-06fe-11eb-9016-075d58d638fc.png)

**Comment by IdaAkiwumi** [comment]
Update: This is also happening on the other main pages. Safe to assume it's on the whole website.

**Comment by kimisgold** [comment]
It's odd to me that WAVE is still flagging it, as screen readers will never see this form due to it having the `visibility: hidden`. [Their own documentation recommends this as the correct way to hide content from all users.](https://webaim.org/techniques/css/invisiblecontent/) As this is markup controlled by an external source, I'm going to let this one go.
